glean-core/ffi/glean.h

```diff
@@ -402,6 +402,17 @@ void glean_destroy_uuid_metric(uint64_t v);
  */
 void glean_enable_logging(void);
 
+/**
+ * Initialize the logging system to send JSON messages to a pipe.
```

Comment:
While that is how we _use_ it we don't really care if that's a pipe, a file descriptor to an actual file or even a tcp socket.
Do you think this is still the right docs & name?

(Given that Python is the only user of this  and there we DO use a pipe this is a weakly held opinion on naming)

Reply:
That's good point.  Can rename.